lib/common/transformGroups.js

```diff
@@ -223,5 +223,51 @@ module.exports = {
    */
   'assets': [
     'attribute/cti'
-  ]
+  ],
+  /**
+   * Transforms:
+   *
+   * [attribute/cti](transforms.md#attributecti)
+   * [name/cti/camel](transforms.md#namecticamel)
+   * [color/hex8flutter](transforms.md#colorhex8flutter)
+   * [size/flutter/remToDouble](transforms.md#sizeflutterremToDouble)
+   * [content/flutter/literal](transforms.md#contentflutterliteral)
+   * [asset/flutter/literal](transforms.md#assetflutterliteral)
+   * [font/flutter/literal](transforms.md#fontflutterliteral)
+   *
+   * @memberof TransformGroups
+   */
+  'flutter': [
+    'attribute/cti',
+    'name/cti/camel',
+    'color/hex8flutter',
+    'size/flutter/remToDouble',
+    'content/flutter/literal',
+    'asset/flutter/literal',
+    'font/flutter/literal'
+  ],
+  /**
+   * Transforms:
+   *
+   * [attribute/cti](transforms.md#attributecti)
+   * [name/ti/camel](transforms.md#nameticamel)
+   * [color/hex8flutter](transforms.md#colorhex8flutter)
+   * [size/flutter/remToDouble](transforms.md#sizeflutterremToDouble)
+   * [content/flutter/literal](transforms.md#contentflutterliteral)
+   * [asset/flutter/literal](transforms.md#assetflutterliteral)
+   * [font/flutter/literal](transforms.md#fontflutterliteral)
+   *
+   * This is to be used if you want to have separate files per category and you don't want the category (e.g., color) as the lead value in the name of the property (e.g., StyleDictionaryColor.baseText instead of StyleDictionary.colorBaseText).
+   *
+   * @memberof TransformGroups
+   */
+  'flutter-separate': [
+    'attribute/cti',
+    'name/ti/camel',
```

Comment:
I like this. Creating 2 transform groups so it is easy for people to generate a single file or multiple files!